guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc

```diff
@@ -30,6 +30,7 @@ Use the `{foreman-installer}` command to configure {Project} to connect to an ex
 
 To enable the Secure Sockets Layer (SSL) protocol for these external databases, add the following options:
 +
+ifdef::foreman-el,katello,foreman-deb[]
```

Comment:
I'd make this the opposite of below so you don't exclude orcharhino
```suggestion
ifndef::satellite[]
```

Reply:
Agree :+1: